nuxeo-core/nuxeo-core-storage-mongodb/src/test/java/org/nuxeo/ecm/core/storage/mongodb/TestMongoDBRepositoryService.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.ecm.core.storage.mongodb;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.function.BiConsumer;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.runtime.model.DefaultComponent;
+import org.nuxeo.runtime.mongodb.MongoDBConnectionConfig;
+import org.nuxeo.runtime.mongodb.MongoDBConnectionService;
+import org.nuxeo.runtime.mongodb.MongoDBFeature;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+/**
+ * @since 11.1
+ */
+@RunWith(FeaturesRunner.class)
+@Features(MongoDBFeature.class)
+public class TestMongoDBRepositoryService {
+
+    @Inject
+    protected MongoDBRepositoryService mongoDBRepositoryService;
+
+    @Test
+    public void testUP() {
+        assertNotNull(mongoDBRepositoryService);
+    }
+
+    /**
+     * @deprecated since 11.1. This test is just for ensuring the correct injection of
+     *             {@link org.nuxeo.ecm.core.storage.mongodb.MongoDBRepositoryService} and the ability to handle the
```

Comment:
I would have put the whole description in the javadoc description itself, or in a comment in the test method. It's quite hard to read there.

Reply:
I took the hypothesis that I had to format the wholes files (java classes or xml) where a class will be impacted as example the class org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService  is impacted in java code. And then I did the xml file 

i will pay attention about the separation and formatting only the affected files only